docs/src/using-turing/guide.md

```diff
@@ -42,7 +42,12 @@ end
 ```
 
 
-Note: As a sanity check, the expectation of `s` is 49/24 (2.04166666...) and the expectation of `m` is 7/6 (1.16666666...).
+Note: As a sanity check, the expectation of `s²` is 3 (theta/(alpha - 1)) and the expectation of `m` is 0. This can be easily checked using Prior:
```

Comment:
theta and alpha are not defined, are they? Maybe an alternative could be
```suggestion
Note: As a sanity check, the prior expectation of `s²` is `3/(2 - 1) = 3` and the prior expectation of `m` is 0. This can be easily checked using `Prior`:
```

Reply:
theta and alpha I took to be implicitly defined by the model. I think your wording is better but maybe it is even clearer if we still use theta and alpha explicitly in the expression as those are the names of the parameters given in the doc.

```suggestion
Note: As a sanity check, the prior expectation of `s²` is 3 (theta/(alpha - 1), where theta is 3 and alpha is 2) and the prior expectation of `m` is 0. This can be easily checked using `Prior:
```

Reply:
Which doc? I couldn't find theta and alpha in the guide, maybe I missed it?

Reply:
Sorry for the confusion, I meant the help for InverseGamma (?InverseGamma).

My argument, in full, for using alpha and theta is the following.

The user sees `InverseGamma(2, 3)`.  The user might not know/remember what the expected value of a random variable with an inverse gamma is.  But upon doing `?InverseGamma` they can see that in this model `alpha = 2, theta = 3`.  And they can go to the Wikipedia page linked from that same help for InverseGamma ---https://en.wikipedia.org/wiki/Inverse-gamma_distribution) that the expression for the mean is theta/(alpha - 1).   (Note that the Wikipedia uses beta instead of theta).

So by giving the value of 3 as an explicit function of alpha and theta (and saying what values alpha and theta are taking), a user can quickly check where that 3 and that 3/(2-1) are coming from.

Reply:
Thank you for the explanatoon! While I can see the motivation for alpha and theta now, I think that their use rather creates confusion. In the guide alpha and theta are not explained and not used in any other place, so readers do not know that they are supposed to refer to the namea lls of the parameters in the docstring of `InverseGamma`.